### DISCUSSION TITLE
Cannot see website in github pages

### DISCUSSION THREAD

**Discussion by Satindar31** [community/community#23835] [Other Feature Feedback, Questions, & Ideas]
<p>I tried Github pages and I published my site. When I visited it I could only see the navbar and my friends can see the whole site. when I try my site in my PC it works fine. This is the website link <a href="https://solarblade.github.io" rel="noopener nofollow ugc">https://solarblade.github.io</a></p>

**Comment by Valenchak** [comment]
<p><a href="https://solarblade.github.io/" rel="noopener nofollow ugc">SolarBlade</a><br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/f/1/f1d69ad1e11ac1b64c914b4ed2822f9a71546b47.jpeg" data-download-href="/uploads/short-url/yvoRfROSHF7QbKtca8bSspoiwgn.jpeg?dl=1" title="Снимок" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/840275/181079640-92a3d16e-3740-4957-921b-8203899c832f.jpeg" alt="Снимок" data-base62-sha1="yvoRfROSHF7QbKtca8bSspoiwgn" width="690" height="280" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/f/1/f1d69ad1e11ac1b64c914b4ed2822f9a71546b47_2_690x280.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/f/1/f1d69ad1e11ac1b64c914b4ed2822f9a71546b47_2_1035x420.jpeg 1.5x, https://aws1.discourse-cdn.com/github/original/3X/f/1/f1d69ad1e11ac1b64c914b4ed2822f9a71546b47.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/f/1/f1d69ad1e11ac1b64c914b4ed2822f9a71546b47_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Снимок</span><span class="informations">1342×546 50.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div><br>
:+1:

**Comment by Satindar31** [comment]
<p>it somehow worked by clearing cache for GitHub pages and my site</p>
